R/calcEDGETrData.R

```diff
@@ -44,7 +44,9 @@ calcEDGETrData <- function() {
       'SSP2EU',    'NAV_ele',                    'default',
       'SSP2EU',    'NAV_all',                    'SSP2EU_lowdem',
```

Comment:
Am besten einmal in allen Paketen nach SSP2EU_lowdem suchen und mit SSP2EU_demRedStrong ersetzen :)